services/drupal/composer.patches.json

```diff
@@ -209,6 +209,9 @@
     "drupal/entitygroupfield":  {
       "Limit available groups in widget to ones user has permission to.": "https://www.drupal.org/files/issues/2020-08-07/entitygroupfield-Autocomplete_permissions-3163814-4.patch"
     },
+    "drupal/environment_indicator": {
+      "Add configurable option to always link to environment front pages.": "https://git.drupalcode.org/project/environment_indicator/-/merge_requests/4.patch"
```

Comment:
Please reference a copy of the patch file locally in the repo rather than referencing the merge request directly.  See this for more info: https://github.com/cweagans/composer-patches/issues/347

Also, please be sure to include the relevant drupal issue ID in either the patchfile name or the patch description.

Reply:
I've committed the patch locally now.